workers/go/throughputstress/workflow.go

```diff
@@ -36,6 +36,9 @@ func ThroughputStressWorkflow(ctx workflow.Context, params *throughputstress.Wor
 		return output, err
 	}
 	err = workflow.SetUpdateHandler(ctx, UpdateSleep, func(ctx workflow.Context) error {
+		if params.SkipSleep {
```

Comment:
I wonder if instead of making an `UpdateSleep` update stop sleeping, you should avoid calling `UpdateSleep` altogether if this flag is set.

Reply:
Yeah, makes sense. Adjusted to do that.